modules/programs/vivid.nix

```
@@ -34,8 +37,15 @@ in
     enableFishIntegration = mkFishIntegrationOption { inherit config; };
 
     colorMode = mkOption {
-      type = with types; nullOr str;
-      default = null;
+      type =
+        with types;
+        nullOr (
+          either str (enum [
+            "8-bit"
+            "24-bit"
+          ])
+        );
+      default = "24-bit";
```

The default should stay `null` to use the CLI's true default:

```suggestion
      default = null;
```